consultation_analyser/support_console/views/consultations.py

```diff
@@ -46,6 +46,24 @@ def delete(request: HttpRequest, consultation_id: UUID) -> HttpResponse:
     return render(request, "support_console/consultations/delete.html", context=context)
 
 
+def get_overall_number_themes_latest_run(consultation):
+    total_themes = 0
+    total_with_summaries = 0
+    processing_run = consultation.latest_processing_run
+
+    if not processing_run:
+        return total_themes, total_with_summaries
+
+    free_text_questions = models.Question.objects.filter(section__consultation=consultation)
+    for question in free_text_questions:
+        qs = processing_run.get_themes_for_question(question.id)
```

Comment:
Could we `processing_run.themes.count()` here?

Reply:
Yes, now updated!